en/guide/rtsp.md

```diff
@@ -0,0 +1,74 @@
+# RTSP Streaming & Discovery
+
+## RTSP Usage Flow
+
+The diagram below shows the common sequence for advertising and accessing RTSP video streams.
+
+<!-- 
+{/% mermaid %}
```

Comment:
Ignore this. It is disabled sequence generator. We could use this instead of the image below.